documentation/guides/migrating-from-v4-to-v5.md

```diff
@@ -0,0 +1,158 @@
+# Migrating from v4 to v5
+
+Polaris v5.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/v5.0.0)) removes most features deprecated in v4.x.x releases. This file describes all code updates required to stay up to date.
+
+## AppBridge integrations removed
+
+In v3 and v4 `AppProvider` could accept `apiKey` `shopOrigin` and `forceRedirect` props that would configure AppBridge delegation in our `ResourcePicker`, `Loading`, `Modal`, `Page` and `Toast` components.
```

Comment:
> In v3 and v4 `AppProvider` could accept `apiKey` `shopOrigin` and `forceRedirect` props that would configure AppBridge delegation in our `ResourcePicker`, `Loading`, `Modal`, `Page` and `Toast` components.

I think we should rephrase this to say that in v5 these props are removed. I know the title above says integrations removed but it's not clear if these are the integrations that are removed or what works in v3 and v4.